### DISCUSSION TITLE
Can someone correct the Vietnamese error or someone who is Vietnamese can correct it

### DISCUSSION THREAD

**Discussion by NgnVhung** [florisboard/florisboard#2421] [Q&A]
![Screenshot_20240316-003121](https://github.com/florisboard/florisboard/assets/155422237/57968f77-c565-408c-af57-62aafbfd914d)

****Vietnamese on my Florisboard keyboard version 0.4.0 Beta 01 has spelling errors
Please****

**Comment by ngocthanhgl** [comment]
Check my repo
